ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -265,39 +250,25 @@ export default class ConfirmTransactionBase extends Component {
       },
     });
 
-    if (process.env.SHOW_EIP_1559_UI) {
-      this.setState({ editingGas: true });
-    } else {
-      showCustomizeGasModal();
-    }
+    this.setState({ editingGas: true });
   }
 
-  // TODO: rename this 'handleCloseEditGas' later when we remove the
-  // SHOW_EIP_1559_UI flag/branch
-  handleCloseNewGasPopover() {
+  handleCloseEditGas() {
     this.setState({ editingGas: false });
   }
 
   renderDetails() {
     const {
-      primaryTotalTextOverride,
```

Comment:
primaryTotalTextOverride and secondaryTotalTextOverride need to remain

Reply:
These were added to the end of the object -- they're still here

Reply:
yes, my mistake!